test/regex.test.js

```diff
@@ -12,10 +12,10 @@ const good = [
   RegExp(Array(26).join('a?') + Array(26).join('a'))
 ]
 
-test('safe regex', function (t) {
+test('safe regex', async t => {
```

Comment:
Not used `node:test` much yet, but do these need to be async? The tests aren't doing anything async.